plugins/woocommerce/src/Internal/DataStores/Orders/CustomOrdersTableController.php

```diff
@@ -586,14 +602,14 @@ private function get_hpos_setting_for_sync() {
 			if ( $is_dangerous ) {
 				$sync_message[] = wp_kses_data(
 					sprintf(
-					// translators: %d: number of pending orders.
+						// translators: %s: number of pending orders.
 						_n(
-							"There's %d order pending sync. <b>Switching data storage while sync is incomplete is dangerous and can lead to order data corruption or loss!</b>",
-							'There are %d orders pending sync. <b>Switching data storage while sync is incomplete is dangerous and can lead to order data corruption or loss!</b>',
+							"There's %s order pending sync. <strong>Switching data storage while sync is incomplete is dangerous and can lead to order data corruption or loss!</strong>",
+							'There are %s orders pending sync. <strong>Switching data storage while sync is incomplete is dangerous and can lead to order data corruption or loss!</strong>',
```

Comment:
Maybewe can extract the common message out, wdyt? For a moment, I was confused as to why we had two similar messages, and then I realized.

Seems duplicate, can be extracted out.
```
<strong>Switching data storage while sync is incomplete is dangerous and can lead to order data corruption or loss!</strong>
```

Reply:
I guess there might be some i18n issues with separating the strings, but I did it anyways in d55c61cd36668f6a787b88ac3392809595bc0082. We can reassess later if this ever becomes a problem for translations, which it most likely won't.